src/components/docs/Cards/BaseContainer/index.js

```diff
@@ -4,14 +4,14 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames';
 import styles from './styles.module.css';
 
-const BaseContainer = ({children, className, hasShadow, hasRoundedCorners, to, ...props}) => (
-  <a 
+const BaseContainer = ({children, className, hasShadow, hasRoundedCorners, to, target = '_self'}) => (
```

Comment:
This was getting passed extra non-anchor props. I simplified to allow a target.

Reply:
If I'm not mistaken, how react would handle this if it no target was passed in and there was no default parameter is that the attribute would be dropped. That seems cleaner to me in those situations because it keeps the markup uncluttered if someone inspects the page. But, it's a nit so it's up to you